apps/jubmoji-quest/prisma/migrations/20231019183722_create_card_quest_power/migration.sql

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
Can we instructions in README for how to set up local PostgreSQL db and initializing it using `prisma migrate dev`?

Reply:
Added prisma instructions